### PR TITLE
ci: enable Dependabot for website, submodules, and GitHub Actions

### PR DESCRIPTION
**Pull request by ylemkimon**
Dependabot turned out to be quite nice at keeping dependencies up to date and pull requests seem to be manageable. This enables it for three more: website, submodules (katex-fonts), and GitHub Actions.

### CODE REVIEW

**Comment by kevinbarabash** [comment]
It'll be interesting to see how many dependabot PRs this results in.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -7,3 +7,17 @@ updates:
...
       interval: "daily"
+  - package-ecosystem: "npm"
+    directory: "/website/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "gitsubmodule"
+    directory: "/" # Location of .gitmodules
+    schedule:
+      interval: "daily"
```
Comment (kevinbarabash) [inline .github/dependabot.yml:18]: Will this try to update the submodules to `master`?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -7,3 +7,17 @@ updates:
...
+
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of .github
```
Comment (kevinbarabash) [inline .github/dependabot.yml:21]: Thanks for adding these comments.

File: .github/dependabot.yml (hunk 1 above)
Comment (ylemkimon) [reply .github/dependabot.yml:18]: @kevinbarabash Yes, it'll.

**Pull request merged**